src/elements/content-sidebar/DetailsSidebar.js

```diff
@@ -48,6 +48,7 @@ type ExternalProps = {
     onClassificationClick?: (e: SyntheticEvent<HTMLButtonElement>) => void,
     onRetentionPolicyExtendClick?: Function,
     onVersionHistoryClick?: Function,
+    refreshIdentity?: boolean,
```

Comment:
Might be cool if this were a function or boolean, so the parent component could pass in a callback for on refresh, maybe in a follow up if useful.

Reply:
yeah will revisit this to see if I can get a programmatic refresh on each Panel itself instead of relying on this toggling refresh prop